### PR TITLE
Enable CI on GitHub Actions

### PR DESCRIPTION
**Pull request by ericonr**
* test description *

### CODE REVIEW

**Comment by Anachron** [comment]
What's needed here to merge this PR? Is there a test-repo where this was successful?

**Comment by ericonr** [comment]
Testing builder power: can it build a kernel?

Will update with new version after this test is over.

**Comment by Anachron** [comment]
Nice, the `i686` build succeeded.

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+      - name: Checkout
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 200
+
+      - name: Create hostrepo
+        run: ln -s "$(pwd)" /hostrepo
+
+      - name: Set mirrors
```
Comment (sgn) [inline .github/workflows/build.yaml:74]: I don't think we need `name: `, the script's name is descriptive enough, without name, it's defaulted to `run: `

File: .github/workflows/build.yaml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        run: |
+          (
+          here="$(pwd)"
+          cd /
+          "$here/common/travis/build.sh" "$BOOTSTRAP" "$ARCH"
+          )
+
+      - name: Show files
+        run: |
+          (
```
Comment (sgn) [inline .github/workflows/build.yaml:96]: subshell is not necessary, I think.

File: .github/workflows/build.yaml [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+      - name: Fetch xtools
+        run: common/travis/fetch-xtools.sh
+      - name: Run xlint
+        run: common/travis/xlint.sh
+
+  # Build changed packages.
+  build:
+    name: Build packages
+    runs-on: ubuntu-18.04
+    if: "!github.event.pull_request.draft && !contains(github.event.pull_request.title, '[ci skip]' && !contains(github.event.pull_request.body, '[ci skip]')"
```
Comment (sgn) [inline .github/workflows/build.yaml:34]: With this `if`, I think the Actions won't build the PR with title or the body (first comment) that has '[ci skip]`, right?

While I like it to not pollute the commit message, I think we should update the README, too.

Comment (ericonr) [reply .github/workflows/build.yaml:34]: I updated it in CONTRIBUTING, I don't think the README mentions it.

File: .github/workflows/build.yaml (hunk 2 above)
Comment (ericonr) [reply .github/workflows/build.yaml:96]: It might be, because of the `cd`...

File: .github/workflows/build.yaml (hunk 1 above)
Comment (ericonr) [reply .github/workflows/build.yaml:74]: Agreed on most. I cleaned it up.

**Comment by ericonr** [comment]
It has stopped working and I have no idea why. Will investigate further tomorrow

**Pull request merged**